tests/locket_tests.py

```diff
@@ -177,10 +177,11 @@ def lock_is_released_if_holding_process_is_brutally_killed(lock_path):
     assert locker_1.has_lock()
     assert not locker_2.has_lock()
     
-    locker_1.kill(signal.SIGKILL)
+    locker_1.kill(getattr(signal, "SIGKILL", signal.SIGTERM))
     time.sleep(0.1)
     
     assert locker_2.has_lock()
+    locker_2.release()
```

Comment:
Note the explicit release() calls are required, otherwise removing the temp directory fails (file is still in use).